src/sql/src/rbac.rs

```diff
@@ -1499,15 +1512,18 @@ pub(crate) const fn all_object_privileges(object_type: SystemObjectType) -> AclM
     }
 }
 
-pub(crate) const fn owner_privilege(object_type: ObjectType, owner_id: RoleId) -> MzAclItem {
+pub const fn owner_privilege(object_type: ObjectType, owner_id: RoleId) -> MzAclItem {
     MzAclItem {
         grantee: owner_id,
         grantor: owner_id,
         acl_mode: all_object_privileges(SystemObjectType::Object(object_type)),
     }
 }
 
-pub(crate) const fn default_builtin_object_privilege(object_type: ObjectType) -> MzAclItem {
+pub const fn default_builtin_object_privilege(
+    object_type: ObjectType,
+    mz_system_role_id: RoleId,
```

Comment:
I'm on the fence about this, but I feel like `MZ_SYSTEM_ROLE_ID` could also get moved to `mz_sql`. Not a huge deal, but would prevent needing to take it as an arg here

Reply:
I moved it, I agree that it's a bit cleaner.